sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableAsyncClientTest.java

```diff
@@ -67,16 +67,18 @@ protected HttpClient buildAssertingClient(HttpClient httpClient) {
 
     protected void beforeTest() {
         final String tableName = testResourceNamer.randomName("tableName", 20);
-        tableClient = getClientBuilder(tableName, true).buildAsyncClient();
+        tableClient = getClientBuilder(tableName, false)
+            .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
+            .buildAsyncClient();
 
-        tableClient.createTable().block(DEFAULT_TIMEOUT);
+        tableClient.createTable().block();
```

Comment:
Is there a particular reason we moved on from using a timeout here? I see it used in every other operation that blocks. That said, I think we should avoid blocking operations when testing async clients and instead use them asynchronously as they would be by a final user. Maybe that can happen in a separate PR.

Reply:
@vcolin7 Nope, this was left out by accident while trying to debug an issue. I'll re-add it.